appveyor.yml

```diff
@@ -0,0 +1,7 @@
+build_script:
```

Comment:
We don't use appveyor, we use TeamCity instead. Are you setting up a CI build for Npgsql or something?